src/groovy/heim/session/SessionContext.groovy

```diff
@@ -19,11 +19,18 @@ class SessionContext {
 
     final String workflowType
 
+    Date lastActive = new Date()
```

Comment:
This variable is written by web worker threads, but read from a separate thread (the collection thread). `ConcurrentMap` (where the session context is stored) only guarantees that "actions in a thread prior to placing an object into [the ConcurrentMap] happen-before actions subsequent to the access [...] of that object from the in another thread". Because `lastActive` is written _after_ the session context is put into the concurrent map, these writes will not be published to the collection thread.